bin/all_sky_search/pycbc_template_kde_calc

```diff
@@ -52,44 +59,50 @@ def optimizedparam(sampleval, nfold=2):
     maxFOM = FOM[(optbw, optalpha)]
     return optbw, optalpha
 
+assert len(args.fit_param) == len(args.log_param)
+
 if args.make_signal_kde and args.make_template_kde:
-    raise Exception("Choose only one option out of --make-signal-kde and --make-template-kde")
+    parser.error("Choose only one option out of --make-signal-kde and --make-template-kde")
```

Comment:
Same for these lines, can you move them to just after argument parsing and `init_logging`